go.mod

```diff
@@ -25,4 +25,5 @@ require (
 	golang.org/x/net v0.0.0-20200301022130-244492dfa37a // indirect
 	golang.org/x/sys v0.0.0-20200302150141-5c8b2ff67527 // indirect
 	gopkg.in/yaml.v2 v2.2.8 // indirect
+	mvdan.cc/gofumpt v0.0.0-20200627213337-90206bd98491 // indirect
```

Comment:
Please remove this from here. You must `go get` outside of the project or use go install to not change the go mode file.